### DISCUSSION TITLE
hide mandatory fields

### DISCUSSION THREAD

**Discussion by Akaki-1991** [YetiForceCompany/YetiForceCRM#15842] [How to] [answered]
hello,

how to hide default mandatory fields?

**Comment by rskrzypczak** [comment]
You can try to use dependent field functionality.
_Software configuration/Standard modules/Dependent Fields_

**Reply by Akaki-1991** [reply]
> 
can i use field type?

**Reply by rskrzypczak** [reply]
no

**Reply by Akaki-1991** [reply]
> no
thank you my friend

**Reply by s2b-git** [reply]
rskrzypczak, mandatory fields seems not working in 6.4
Checked on few scenarios and no result as expected

Best
![image](https://user-images.githubusercontent.com/84110093/205052471-61ca4d14-2ffb-4d31-9355-592fd92e91a4.png)

**Reply by s2b-git** [reply]
I am sorry... it does but with only one value from multiselect field. Apologize.
Maybe will be OK to add to condition "Contains" ?

**Comment by s2b-git** [comment]
https://gitdeveloper.yetiforce.com/index.php?parent=Settings&module=FieldsDependency&view=Edit&record=2

**Comment by Jianbin719** [comment, marked as answer]
Try to remove the field from "public $mandatory_fields = " in related module file, like Yetiforce/modules/Accounts/Accounts.php.

But sometimes, the error comes.
